### ISSUE TITLE
Incorrect versions in .env for tag 8.6-alpha4

### ISSUE THREAD

**Issue by schattenk** [camunda/camunda-platform#784]
See https://github.com/camunda/camunda-platform/blob/8.6-alpha4/.env

The version numbers are incorrect. Expected: 8.6-alpha4, Actual: 8.5.3, 8.5.2

I was wondering why the new features didn't work. That was the explanation.

**Comment by jessesimpson36** [comment]
We have an internal task that will fix this problem https://github.com/camunda/distribution/issues/288 and we will close this issue when the other issue is closed.

**Comment by Arpit529Srivastava** [comment]
@schattenk @jessesimpson36 i can work on this issue please assign it to me and add it under the label ```hacktoberfest```

**Comment by jessesimpson36** [comment]
I don't think this is a good task to work on for hacktoberfest. main has correct versions inside the `.env` file. And tags cannot be changed inside of a PR.

If you have code recommendations for how we can change our `.github/workflows` so that a release does not go through unless `.env` is updated, I would be open for that sort of change.  but `.env` already automatically updates  as long as there is not a bug in our renovate config.

**Comment by hisImminence** [comment]
As this repo was is deprecated and docker compose was moved to [camunda-self-managed](https://github.com/camunda/camunda-self-managed/tree/main/docker-compose), this ticket can be closed.
